data/dataset_generation.ipynb

```diff
@@ -292,6 +292,31 @@
     "with open(\"spambrainz_dataset_eval.pickle\", \"wb\") as f:\n",
     "    pickle.dump(eval_set, f)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Prediction set\n",
+    "\n",
+    "PREDICT_COUNT = 100 + EVAL_COUNT\n",
+    "\n",
+    "eval_set = np.empty(((PREDICT_COUNT - EVAL_COUNT)*2, 524))\n",
+    "\n",
+    "i = 0\n",
+    "j = 1\n",
+    "for key in spam_keys[EVAL_COUNT:PREDICT_COUNT]:\n",
+    "    eval_set[i] = preprocess_editor(spam_editors[key], 1)\n",
+    "    i += 2\n",
+    "for key in nonspam_keys[EVAL_COUNT:PREDICT_COUNT]:\n",
+    "    eval_set[j] = preprocess_editor(nonspam_editors[key], 0)\n",
+    "    j += 2\n",
+    "\n",
+    "with open(\"spambrainz_dataset_predict.pickle\", \"wb\") as f:\n",
```

Comment:
Please update script to download `spambrainz_dataset_predict.pickle`.

Reply:
The script already contains spambrainz_dataset_predict.pickle